ui/js/directives/treeherder/clonejobs.js

```diff
@@ -526,29 +566,29 @@ treeherder.directive('thCloneJobs', [
 
             //Empty the job column before populating it
             jobTdEl.empty();
-
-            var jgObj, jobGroup, i;
+            var jgObj, jobGroup, i, jobTdHtml = "";
             for (i=0; i<jobGroups.length; i++) {
 
                 jgObj = jobGroups[i];
-
                 if (jgObj.symbol !== '?') {
                     // Job group detected, add job group symbols
                     jobGroups[i].grkey = jgObj.mapKey;
                     jobGroups[i].collapsed = true;
-                    jobGroup = $(jobGroupInterpolator(jobGroups[i]));
-                    jobTdEl.append(jobGroup);
-
                     if (isGroupExpanded(jgObj)) {
-                        addJobBtnEls(jgObj, jobGroup.find(".group-job-list"));
+                        jobGroups[i].btnHTML = getJobBtnEls(jgObj);
```

Comment:
Instead of assigning to temporaries and then interpolating, I'd just call the interpolator directly.
i.e.
```js
jobTdHtml += jobGroupInterpolator({ btnHTML: getJobBtnEls(jgObj), ... })
```

Ditto below.